sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -917,15 +917,45 @@ BOOT_POLICY_VALID_CASES = [
     for b in BOOT_POLICY_VALID_CASES
 ]
 
+[
+    otp_image(
+        name = "otp_img_boot_policy_valid_{}".format(lc_state.lower()),
+        src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state.lower()),
+        overlays = [
+            "//hw/ip/otp_ctrl/data:otp_json_creator_sw_cfg",
+            "//hw/ip/otp_ctrl/data:otp_json_owner_sw_cfg",
+            "//hw/ip/otp_ctrl/data:otp_json_hw_cfg",
+        ],
+        tags = ["vivado"],
```

Comment:
No need for `vivado` here. I think this is pretty fast, so not much to be gained by marking it manual, either :)

Reply:
Yes, indeed. This is my early morning copy-pasta with fresh mozzarella :)

P.S. I wish the CI was a little bit faster so that I wouldn't need to make the same fix over and over again.